### ISSUE TITLE
Player height binds?

### ISSUE THREAD

**Issue by marcocorriero** [matzman666/OpenVR-AdvancedSettings#85]
Player height binds?

I would be amazing when i play a game, and i want to trew a hold object out from the VR map

**Comment by matzman666** [comment]
I don't quite understand your request. You can already adjust the player height and add a button to toggle the adjustment on the [Accessibility page](https://github.com/matzman666/OpenVR-AdvancedSettings#--accessibility-page), or do you mean something different?

**Comment by marcocorriero** [comment]
yeah, bind a specific height to keyboard, so u can press when you are playing without interrupting the game.

**Comment by StewieGriffin** [comment]
Quite frankly, this seems a good idea, but there is a lot of potential for abuse from cheaters (in VR multiplayer shooters, for example)

**Comment by marcocorriero** [comment]
Some VR games doesn't let you set the height, i  want to drop like in gorn gornies from a huge height,and i need binds to do it.
